lib/sda/sda.go

```diff
@@ -0,0 +1,20 @@
+package sda
+
+/*
+SDA is the Secure Distributed API which offers a simple framework for generating
+your own distributed systems. It is based on a description of your protocol
```

Comment:
nitpicking: 'a simple framework for generating your own distributed systems' is a litte to general, isn't it?